### PR TITLE
[Bug][stream-vec-load] Null data load do not skip the same place data

### LINKED ISSUE #9355: [Bug] [Vectorized][Load] Null data load do not skip the same place data
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master

### What's Wrong?

suite **aggregate.groovy**: group=default,query,query/aggregate, name=aggregate
2022-05-03 22:53:20.936 INFO [main] (RegressionTest.groovy:265) - Some suites failed.

After enable the vec stream load.

### What You Expected?

pass all run-regression-test

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by HappenLee**
# Proposed changes

Issue Number: close #9355

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes)
2. Has unit tests been added: (Yes)
3. Has document been added or modified: (No Need)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (Yes)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/olap/rowset/segment_v2/column_writer.cpp [hunk 1]
```diff
@@ -193,23 +193,43 @@ Status ColumnWriter::append_nullable(const uint8_t* is_null_bits, const void* da
...
+
+    return Status::OK();
+}
+
 Status ColumnWriter::append(const uint8_t* nullmap, const void* data, size_t num_rows) {
     assert(data && num_rows > 0);
+    const auto* ptr = (const uint8_t*)data;
     if (nullmap) {
-        size_t bitmap_size = BitmapSize(num_rows);
-        if (_null_bitmap.size() < bitmap_size) {
```
Comment (Gabriel39) [inline be/src/olap/rowset/segment_v2/column_writer.cpp:200]: `_null_bitmap` is not used after this PR. We should remove this variable from ColumnWriter

**Review by Gabriel39 — CHANGES_REQUESTED** [review]

File: be/src/olap/rowset/segment_v2/column_writer.cpp (hunk 1 above)
Comment (HappenLee) [reply be/src/olap/rowset/segment_v2/column_writer.cpp:200]: the not vectorized load still use `_null_bitmap`, we will remove it after replace the old load way

**Comment by Gabriel39** [comment]
LGTM

**Comment by spaces-X** [comment]
LGTM

**Review by wangbo — APPROVED** [review]
LGTM

**Pull request merged**